data/global/excel/monequip.txt

```diff
@@ -43,9 +43,7 @@ shadowmaster		0	7ar	rarm	3	7xf	rarm	3	7qr	rarm	3	0
 shadowmaster		0	7cs	larm	3	7tw	larm	3	7qr	larm	3	0
 shadowmaster		0		tors	3							0
 shadowmaster		0		head	3							0
-fenris		0	vf3	tors	7							0
 necroskeleton		0	vf3	tors	7							0
 claygolem		0	vf3	tors	7							0
 bloodgolem		0	vf3	tors	7							0
-druidbear		0	vf3	tors	7							0
```

Comment:
This doesn't break anything? Odd. Neat but odd.

Reply:
Monequip.txt basically says "druidbear" gets unique ID "vf3" as a chest piece at lvl1. This is the Summon Armor splash chest

This was replaced with the shockwave on-hit effect in the rework so he doesn't need both.